frontend/src/components/pages/RegistrantExperience/RegistrationSteps/ReviewRegistration/index.tsx

```diff
@@ -1,65 +1,129 @@
 import React from "react";
-import { Box } from "@chakra-ui/react";
+import {
+  Accordion,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  Text,
+} from "@chakra-ui/react";
 import PaymentSummary from "./PaymentSummary";
 import { CartItem } from "../../../../../types/RegistrationTypes";
-import ReviewInformation from "./ReviewInformation";
-
-type ReviewRegistrationProps = {
-  isChecked: boolean;
-  toggleChecked: () => void;
-  campName: string;
-};
+import EditCamperCard from "./EditCamperCard";
+import EditContactCard from "./EditContactCard";
+import { CampResponse } from "../../../../../types/CampsTypes";
+import { RegistrantExperienceCamper } from "../../../../../types/CamperTypes";
+import { usePersonalInfoDispatcher } from "../PersonalInfo/personalInfoReducer";
+import GeneralAccordionButton from "../../../../common/GeneralAccordionButton";
 
 const items: CartItem[] = [
   {
-    name: "yes",
+    name: "yes1",
     campers: 2,
     totalPrice: 15.5,
     details: "Some sample details",
   },
   {
-    name: "yes",
+    name: "yes2",
     campers: 2,
     totalPrice: 15.5,
   },
   {
-    name: "yes",
+    name: "yes3",
     campers: 2,
     totalPrice: 15.5,
   },
   {
-    name: "yes",
+    name: "yes4",
     campers: 2,
     totalPrice: 15.5,
   },
   {
-    name: "yes",
+    name: "yes5",
     campers: 2,
     totalPrice: 15.5,
   },
   {
-    name: "yes",
+    name: "yes6",
     campers: 2,
     totalPrice: 15.5,
   },
   {
-    name: "yes",
+    name: "yes7",
     campers: 2,
     totalPrice: 15.5,
   },
 ];
 
+type ReviewRegistrationProps = {
+  camp: CampResponse;
+  campers: RegistrantExperienceCamper[];
+  setCampers: React.Dispatch<
+    React.SetStateAction<RegistrantExperienceCamper[]>
+  >;
+  reviewSummary: boolean;
+};
+
 const ReviewRegistration = ({
-  isChecked,
-  toggleChecked,
-  campName,
+  camp,
+  campers,
+  setCampers,
+  reviewSummary,
 }: ReviewRegistrationProps): React.ReactElement => {
-  return (
+  const dispatchPersonalInfoAction = usePersonalInfoDispatcher(setCampers);
+
+  return !reviewSummary ? (
     <Box>
-      <PaymentSummary campName={campName} items={items} />
+      <Text
```

Comment:
Could we extract this into a new component called `ReviewInformation`. Then we can use the `frontend/src/components/pages/RegistrantExperience/RegistrationSteps/ReviewRegistration/index.tsx` component as a wrapper component and conditionally render the ReviewInformation or PaymentSummary page depending on the state. 

Right now, it's a bit messy that Payment summary is its own component but review info is not

Reply:
Done, I thought it'd be a good idea to make a folder called `ReviewInformation` under `ReviewRegistration`, and this information folder would have all of the edit cards/components in it.